VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -75,6 +84,19 @@ export class JudgeHearingListComponent implements OnInit, OnDestroy {
       });
   }
 
+  postJudgeEvent() {
```

Comment:
what is the use of the cache here?

Reply:
On the hearing list to raised event for the judge I need to know conderenceId and participantId, I could not pass it as params with router, so I use cache